server/cmwell-cons/app/scripts/templates/zoo.cfg

```diff
@@ -3,4 +3,5 @@ dataDir={{dataDir}}
 clientPort=2181
 initLimit=5
 syncLimit=2
+maxClientCnxns=0
```

Comment:
we tested with 500 which seems more than enough and it wansn't enough.
setting max connections to unlimited will cause the machine socket to be finished